data/ignition.template

```diff
@@ -100,6 +100,24 @@ storage:
       contents:
         inline: |
           PasswordAuthentication yes
+    - path: /etc/systemd/user-generators/100-skopeo.sh
+      mode: 0775
+      contents:
+        inline: |
+          #!/usr/bin/bash
+          IMAGE_STORAGE=/usr/share/assisted-migrations/images
+          if ! command -v skopeo 2>&1 >/dev/null
+          then
+            echo "skopeo could not be found"
+            exit 1
```

Comment:
I wonder about the return code, will the boot will proceed in case of non-zero code?

Reply:
I don't know. I'll change it